app/models/post.rb

```diff
@@ -1,3 +1,15 @@
 class Post < ApplicationRecord
-  belongs_to :author
+  belongs_to :author, class_name: 'User'
+  # , counter_cache: :posts_counter
```

Comment:
- [ ] Kindly remove commented code on line 3 to improve readability and avoid confusion